Sources/searchable-collection-vc/SearchableCollectionViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SearchableCollectionViewController.swift
+//  Demo
+//
+//  Created by Yuliia Veresklia on 20/04/2017.
+//
+//
+
+import Foundation
+import UIKit
+
+class SearchableCollectionController: UIViewController {
+    
+    lazy var collectionView: UICollectionView = {
+        let layout = SearchableCollectionViewLayout()
+        
+        let view = UICollectionView(frame: CGRect.zero, collectionViewLayout: layout)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        (view as UIScrollView).delegate = self
+        view.backgroundColor = UIColor.white
+        view.contentInset = UIEdgeInsetsMake(0.0, 0.0, SearchBarView.height, 0.0)
```

Comment:
We prefer using `UIEdgeInsets(top: CGFloat, left: CGFloat, bottom: CGFloat, right: CGFloat)` instead of `UIEdgeInsetsMake(CGFloat, CGFloat, CGFloat, CGFloat)`.